tests/suite/cli_paths.rs

```diff
@@ -410,43 +410,40 @@ mod windows {
     async fn install_uninstall_affect_path_with_non_unicode() {
         use std::ffi::OsString;
         use std::os::windows::ffi::OsStrExt;
-        use winreg::enums::{RegType, HKEY_CURRENT_USER, KEY_READ, KEY_WRITE};
-        use winreg::{RegKey, RegValue};
+        use windows_registry::{CURRENT_USER, HSTRING};
 
         let mut cx = CliTestContext::new(Scenario::Empty).await;
         let _guard = RegistryGuard::new(&USER_PATH).unwrap();
+
         // Set up a non unicode PATH
-        let reg_value = RegValue {
-            bytes: vec![
-                0x00, 0xD8, // leading surrogate
-                0x01, 0x01, // bogus trailing surrogate
-                0x00, 0x00, // null
-            ],
-            vtype: RegType::REG_EXPAND_SZ,
-        };
-        RegKey::predef(HKEY_CURRENT_USER)
-            .open_subkey_with_flags("Environment", KEY_READ | KEY_WRITE)
-            .unwrap()
-            .set_raw_value("PATH", &reg_value)
+        let bytes = [
+            0x00, 0xD8, // leading surrogate
+            0x01, 0x01, // bogus trailing surrogate
+            0x00, 0x00, // null
+        ];
+        let key = CURRENT_USER.create("Environment").unwrap();
+        key.set_hstring("PATH", &HSTRING::from_wide(&bytes).unwrap())
```

Comment:
I guess `HSTRING::from_wide()` might eliminate the last two `0x00` and modify something, so the CI test will be failed.